Cargo.toml

```diff
@@ -1,19 +1,7 @@
 [workspace]
 resolver = "2"
 exclude = ["libs/nearcore"]
-members = [
-    "crates/attestation", 
-    "crates/contract",
-    "crates/primitives", 
-    "crates/tee_authority", 
-    "crates/test_utils", 
-    "crates/tls",
-    "crates/contract_history",
-    "crates/test_migration_contract",
-    "crates/test_parallel_contract",
-    "crates/devnet",
-    "crates/node",
-]
+members = ["crates/*"]
```

Comment:
I prefer having these explicitly spelled out. Is this inconvenient in any way?

When using `cargo new` it typically auto-adds these anyway.